virttest/env_process.py

```diff
@@ -526,7 +526,13 @@ def validate_memory_resource():
         if (params.get("setup_ksm") == "yes" and
                 params.get("ksm_run", "1") == "1"):
             magnification = 1.2
-        free_mem = "%s KB" % memory.freememtotal()
+        # Test whether is MemAvailable already introduced
+        cmd = "grep -q MemAvailable /proc/meminfo"
```

Comment:
You could do the same with

    'MemAvailable' in genio.read_file('/proc/meminfo')